pkg/virtctl/expose/expose.go

```diff
@@ -88,6 +107,60 @@ func (o *Command) RunE(cmd *cobra.Command, args []string) error {
 	// second argument must be name of the VM
 	vmName := args[1]
 
+	if value, ok := os.LookupEnv("KUBECTL_PLUGINS_LOCAL_FLAG_NAME"); ok {
+		serviceName = value
+	}
+
+	if serviceName == "" {
+		return fmt.Errorf(`required flag "name"`)
+	}
+
+	if value, ok := os.LookupEnv("KUBECTL_PLUGINS_LOCAL_FLAG_PORT"); ok {
+		i64, err := strconv.ParseInt(value, 10, 32)
+		if err != nil {
+			return err
+		}
+
+		port = int32(i64)
+	}
+
+	if port == 0 {
+		return fmt.Errorf(`required flag "port"`)
+	}
+
+	if value, ok := os.LookupEnv("KUBECTL_PLUGINS_LOCAL_FLAG_EXTERNAL_IP"); ok {
+		externalIP = value
+	}
+
+	if value, ok := os.LookupEnv("KUBECTL_PLUGINS_LOCAL_FLAG_LOAD_BALANCER_IP"); ok {
```

Comment:
I think we can solve this differently, in a less error-prone way directly after commandline parsing.

Here the pseudo code:

```golang
	o.cmd.LocalFlags().VisitAll(func(flag *pflag.Flag) {
		env := plugins.FlagToEnv(flag,"KUBECTL_PLUGINS_LOCAL_FLAG"))
                if value, ok := os.LookupEnv(env); ok {
                  flag.Value.Set(value)
                }
	})
```

You can execute this as part of the `RunE` command before the actual command is started.